backend/src/main/java/com/woowacourse/levellog/team/dto/query/AllTeamDetailQueryResults.java

```diff
@@ -0,0 +1,69 @@
+package com.woowacourse.levellog.team.dto.query;
+
+import com.woowacourse.levellog.team.domain.SimpleParticipant;
+import com.woowacourse.levellog.team.domain.SimpleParticipants;
+import com.woowacourse.levellog.team.domain.TeamStatus;
+import com.woowacourse.levellog.team.dto.response.ParticipantResponse;
+import com.woowacourse.levellog.team.dto.response.TeamDetailResponse;
+import com.woowacourse.levellog.team.dto.response.TeamResponse;
+import com.woowacourse.levellog.team.dto.response.WatcherResponse;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AllTeamDetailQueryResults {
+
+    List<AllTeamDetailQueryResult> results;
+
+    public AllTeamDetailQueryResults(final List<AllTeamDetailQueryResult> results) {
+        this.results = results;
+    }
+
+    public boolean isEmpty() {
+        return results.isEmpty();
+    }
+
+    public TeamDetailResponse toResponses(final Long memberId, final LocalDateTime time) {
```

Comment:
toResponse
```suggestion
    public TeamDetailResponse toResponse(final Long memberId, final LocalDateTime time) {
```
TeamDetailResponse 타입으로 변하니 위 네이밍이 적절해보이네요. 😊